### ISSUE TITLE
Support for Flutter Web Version

### ISSUE THREAD

**Issue by grrrgpratik** [chinloyal/pusher_client#17]
I tested the app in the web version and found it does not have support for it. Any plans to add support for the web version in flutter?

**Comment by chinloyal** [comment]
Yes there are plans to add web support, a pull request is always welcome.
